### PR TITLE
Adds a mapping section to the changelog credits.

### PR DESCRIPTION
**Pull request by PJB3005**
Currently @Probe1, @IratePirate, @dylanstrategie and @Duny- are on it.

If anybody thinks they deserve to be on there tell me and I'll add you.

### CODE REVIEW

**Comment by CptWad** [comment]
I guess me
@xpcybic
@MrSegi
@PJB3005
and @d3athrow

**Comment by clusterfack** [comment]
Instead of thanking them let's just round them up and shoot them

**Comment by CptWad** [comment]
alright ill take the bullet first

**Comment by Probe1** [comment]
you wouldn't shoot me, I subtyped the SME nitrogen floors like you wanted.

**Comment by 9600bauds** [comment]
@Blithering should be in as spritedude

**Comment by Probe1** [comment]
@PJB3005 

Alphabetical order please.  Who really goes first?  Me because my map PRs made you think of adding credit?  IratePirate for porting meta?  Pomf for his eternal suppot?  Duny for Defficiency?  

Alphabetical is cleaner.

**Comment by 9600bauds** [comment]
Alphabetical order also puts me first so that's proof that it works

**Comment by Probe1** [comment]
All part of the plan.

File: html/templates/header.html [hunk 1]
```diff
@@ -62,7 +62,7 @@
 					Mapping:
 				</dt>
 				<dd>
-					Probe1, IratePirate, dylanstrategie, Duny
+					Probe1, IratePirate, dylanstrategie, Duny, Pomf123, CptWad, MrSergi, PJB3005, xpcybic
```
Comment (MrSegi) [inline html/templates/header.html]: Spelling mistake: MrSegi not MrSergi.

**Comment by PJB3005** [comment]
Alright typo fixed and sorted alphabetically.

**Comment by Probe1** [comment]
@PJB3005 

aDD @ComicIronic

**Pull request merged**